Joystick/SenseHat.Joystick/SenseHat.Joystick.psm1

```diff
@@ -0,0 +1,42 @@
+#####
+# Private functions
+
+Function GetSenseHatJoystick {
```

Comment:
Shouldn't be Get-SenseHatJoystick (with the "-")

Reply:
Internal function, so i guess thats a no? Unless we should make it a public function.
It simply enumerates all available input devises and finds the one matching the sensehat joystick, so i dont really see a usecase for it as a public function.